src/taskscontroller.py

```diff
@@ -0,0 +1,87 @@
+from logging import getLogger
+from typing import Tuple, List, Union
+
+from apscheduler.schedulers.blocking import BlockingScheduler
+from apscheduler.triggers.cron import CronTrigger
+import yaml
+
+from src.tasks import TaskWrapper
+from config.settings import *
+
+logger = getLogger("general")
+scheduler = BlockingScheduler()
+scheduler.configure(**SCHEDULER)
+
+
+class TasksController:
+    """A class that parses a configuration file,
+    sets tasks to be executed using apscheduler"""
+
+    def get_tasks(self) -> Union[List[Tuple[TaskWrapper, Dict]], None]:
+        """Parses the task file, wraps it in a wrapper
+        class, returns a list with
+        tuples that contain the TaskWrapper and schedule"""
+
+        with open(TASKS_FILE_PATH, "r") as file:
```

Comment:
`file` is ok for variable name, however several editors highlight it specifically. confuses...